File: Examples/src/programmersguide/workingwithpresentation/convertingtopdf/java/ConvertingToPDF.java
/* 
 * Copyright 2001-2013 Aspose Pty Ltd. All Rights Reserved.
 *
 * This file is part of Aspose.Slides. The source code in this file
 * is only intended as a supplement to the documentation, and is provided
 * "as is", without warranty of any kind, either expressed or implied.
 */
 
package programmersguide.workingwithpresentation.convertingtopdf.java;

import com.aspose.slides.*;

public class ConvertingToPDF
{
    public static void main(String[] args) throws Exception
    {
        // The path to the documents directory.
        String dataDir = "src/programmersguide/workingwithpresentation/convertingtopdf/data/";

        //Instantiate a Presentation object that represents a presentation file
        Presentation pres = new Presentation(dataDir+ "Aspose.pptx");


        //Save the presentation to PDF with default options
        pres.save(dataDir+ "demoDefault.pdf", SaveFormat.Pdf);
        System.out.println("Program executed successfully");
    }
}

File: Plugins/Aspose_Slides_Java_for_Ruby/lib/asposeslidesjava/Presentation/convertingtopdf.rb
module Asposeslidesjava
  module ConvertingToPdf
    def initialize()
        data_dir = File.dirname(File.dirname(File.dirname(File.dirname(__FILE__)))) + '/data/'
                
        # Instantiate a Presentation object that represents a PPTX file
        pres = Rjb::import('com.aspose.slides.Presentation').new(data_dir + "Aspose.pptx")

        # Saving the PPTX presentation to Pdf format
        save_format = Rjb::import('com.aspose.slides.SaveFormat')
        pres.save(data_dir + "Aspose.pdf", save_format.Pdf)

        puts "Document has been converted, please check the output file."
    end
  end
end
